### PR TITLE
Khal 0.7.0

### PR DESCRIPTION
**Pull request by benjaminfrank**

### CODE REVIEW

File: Library/Formula/khal.rb [hunk 1]
```diff
@@ -89,14 +89,15 @@ class Khal < Formula
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    version = Language::Python.major_minor_version "python"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{version}/site-packages"
```
Comment (mikemcquaid) [inline Library/Formula/khal.rb]: What does this change fix?

Comment (benjaminfrank) [inline Library/Formula/khal.rb]: Nothing really.
I had some strange runtime error, so I wanted to try if the same error comes up with python3 (it did as it was a config issue on my end).
however, python2 support might be dropped in future, so i copied this snipped from my vdirsyncher.rb which uses python3.

Comment (mikemcquaid) [inline Library/Formula/khal.rb]: Let's just leave it as-is for now as it only supports a single Python version.

Comment (benjaminfrank) [inline Library/Formula/khal.rb]: How can I support multiple python versions? The tool runs under both and something is (slightly) better than python3?

Comment (DomT4) [inline Library/Formula/khal.rb]: [This sort of thing](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/z3.rb).

Comment (benjaminfrank) [inline Library/Formula/khal.rb]: Nice thanks. I'll push a python2 version for now. maybe next release I'll put in both.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/e3ea87bbb264ef1989fe81b9d40a433c992c94f5. Thank you for another Homebrew contribution @benjaminfrank! :smiley_cat:

**Pull request closed without merging**